tutorcodejail/templates/codejail/k8s/hpa.yml

```diff
@@ -0,0 +1,32 @@
+{%- if CODEJAIL_ENABLE_HPA %}
+apiVersion: autoscaling/v2
```

Comment:
I remember you said there was a problem with the API v2 for HPA during Drydock Palm support. Doesn't it apply here?

Reply:
Yes, i'll fail here too because ArgoCD version of prod is older. i'll change it